Kiosk/Bid Fulfillment/BidCheckingNetworkModel.swift

```diff
@@ -22,10 +22,31 @@ class BidCheckingNetworkModel: NSObject {
         self.fulfillmentController = fulfillmentController
     }
 
-    func waitForBidResolution () -> RACSignal {
-        return self.pollForUpdatedSaleArtwork().then { [weak self] (_) in
-            return self == nil ? RACSignal.empty() : self!.checkForMaxBid()
-
+    func waitForBidResolution (bidderPositionId: String) -> RACSignal {
+        return self.pollForUpdatedBidderPosition(bidderPositionId).then { [weak self] in
+            if let me = self {
```

Comment:
Cool – like `me` here. Could be a `guard let me = self else { return RACSignal.empty() }` to avoid indentation.